Sources/Kitura/SSLConfig.swift

```diff
@@ -17,63 +17,52 @@
 import SSLService
 
 // MARK: SSLConfig
-
+/// SSLConfig
 public struct SSLConfig {
 
     // MARK: Properties
-
+    /// Config required to configure the `SSLService`
     public private(set) var config: SSLService.Configuration
 
     // MARK: Lifecycle
 
     #if os(Linux)
-    ///
+    //MARK: Linux
     /// Initialize an SSLService.Configuration instance using a `CA Certificate` file.
     ///
-    /// - Parameters:
-    ///		- caCertificateFilePath:	Path to the PEM formatted CA certificate file.
-    ///		- certificateFilePath:		Path to the PEM formatted certificate file.
-    ///		- keyFilePath:				Path to the PEM formatted key file. If nil, `certificateFilePath` will be used.
-    ///		- selfSigned:				True if certs are `self-signed`, false otherwise. Defaults to true.
-    ///
+    /// - Parameter caCertificateFilePath: Path to the PEM formatted CA certificate file.
```

Comment:
The declaration, parameters and return value are not being displayed for this function.  The same is true for the other function marked under 'Linux' below this one.

Reply:
This is because jazzy uses xcodebuild to generate the docs, and this bit of code isn't reached when this runs as it's for Linux. Issue documented here: https://github.com/realm/jazzy/issues/750.

Reply:
Thanks for the clarification.